README.md

```diff
@@ -1,15 +1,20 @@
 # Personal-Portfolio
-A Demo web page of  my self
+A Demo web page of  my self (Mobile/Desktop version)
 
-> My first project
-Designed a personal webpage about myself with a template from Figma.
+> Figma clone project
+Designed a personal Mobile and Desktop responsive webpage about myself with a template from Figma.
 
-![screenshot](portfolio.png)
+![screenshot](desktop.png)
+![screenshot](mobile.png)
```

Comment:
- [ ] [OPTIONAL] it seems that these images are broken in the README, can you please fix this? 👍 
- [ ] [OPTIONAL] You can also update your demo link in to reflect the actual changes 😉